src/ppu.cpp

```diff
@@ -2429,7 +2428,5 @@ int FCEUX_PPU_Loop(int skip) {
 	}
 
 finish:
-	FCEU_PutImage();
-

```

Comment:
http://tasvideos.org/forum/viewtopic.php?p=478694#478694

> GUI is updated more frequently than on frame boundaries. Registering a function that way allows it to be ran while the emu is paused.

Do scripts that are supposed to work while paused still work?